### DISCUSSION TITLE
How to deal with media type parameters?

### DISCUSSION THREAD

**Discussion by ralfhandl** [OAI/sig-moonwalk#163] [Ideas]
Media types may allow/require media type parameters ([RFC6838](https://datatracker.ietf.org/doc/html/rfc6838)), one prominent example is the `charset` parameter for text media types.

How can it be expressed that a request may support/require certain parameters and parameter values, or that a response will/may produce certain parameters and parameter values?

**Comment by handrews** [comment]
See also:
* #108 
* OAI/OpenAPI-Specification#2342
* OAI/OpenAPI-Specification#2458

**Reply by handrews** [reply]
Also, this is on the list for 3.3.
